### PR TITLE
Ukrainian Translation - Page Weight Chapter

### PR DESCRIPTION
**Pull request by tymosh**
Makes progress on #1873

### CODE REVIEW

File: src/content/uk/2020/page-weight.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+translators: [tymosh]
+henrihelvetica_bio: Henri - розробник-фрілансер, який перетворив свої захоплення на попурі з інженерії продуктивності, приправленого дрібкою користувацького доступу. У вільний від читання щоденного потоку дослідницьких документів і тематичних досліджень або не систематизованого аудиту сайтів у Dev Tools час, Henri робить внесок у спільноту, приймає участь у зустрічах з групового програмування, включаючи <a href="https://twitter.com/towebperf">Toronto Web Performance Group</a> або добровільно віддає свій обідній час і навчається у всіляких таборах. В інший час він бавиться з програмним забезпечення для створення музики, або найімовірніше тренується і фокусується на тому, щоб здолати найшвидше 5-кілометрівку.
+discuss: 2054
+results: https://docs.google.com/spreadsheets/d/1wG4u0LV5PT9aN-XB1hixSFtI8KIDARTOCX0sp7ZT3h0/
+featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію: рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```
Comment (tunetheweb) [inline src/content/uk/2020/page-weight.md:13]: This is your problem. You can't use colons in YAML as they mean something special so have to use the HTML entity instead:

```suggestion
featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію:&colon; рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```

After that it seems to generate fine.

**Comment by tymosh** [comment]
@tunetheweb thanks! Page is ready then.

File: src/content/uk/2020/page-weight.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+henrihelvetica_bio: Henri - розробник-фрілансер, який перетворив свої захоплення на попурі з інженерії продуктивності, приправленого дрібкою користувацького доступу. У вільний від читання щоденного потоку дослідницьких документів і тематичних досліджень або не систематизованого аудиту сайтів у Dev Tools час, Henri робить внесок у спільноту, приймає участь у зустрічах з групового програмування, включаючи <a href="https://twitter.com/towebperf">Toronto Web Performance Group</a> або добровільно віддає свій обідній час і навчається у всіляких таборах. В інший час він бавиться з програмним забезпечення для створення музики, або найімовірніше тренується і фокусується на тому, щоб здолати найшвидше 5-кілометрівку.
+discuss: 2054
+results: https://docs.google.com/spreadsheets/d/1wG4u0LV5PT9aN-XB1hixSFtI8KIDARTOCX0sp7ZT3h0/
+featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію:&colon; рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```
Comment (tunetheweb) [inline src/content/uk/2020/page-weight.md:13]: Forgot to remove the original colon - oops!

```suggestion
featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію&colon; рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
```

File: src/content/uk/2020/page-weight.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
... він був не важливим, коли запустився базований на тексті [Craigslist](https://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру [оригінальної Doom](https://www.wired.com/2016/04/average-webpage-now-size-original-doom/). Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи [нашу добру знайому Tammy Everts](https://www.speedcurve.com/blog/web-performance-page-bloat/), але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на [вивчені десятиліття тому уроки](https://blog.chriszacharias.com/page-weight-matters), ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```
Comment (tunetheweb) [inline src/content/uk/2020/page-weight.md:29]: We actually started to external links (except Twitter, Wikipedia and MDN as they are often available in translations). Guessing you started this translation before then as the later ones seem to be done!

```suggestion
#РозмірСторінкиВсеЩеВажливий майже натякає на те, що розмір сторінки не важливий, або ніколи не був. Можливо, він був не важливим, коли запустився базований на тексті [Craigslist](https://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру <a hreflang="en" href="https://www.wired.com/2016/04/average-webpage-now-size-original-doom/">оригінальної Doom</a>. Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи <a hreflang="en" href="https://www.speedcurve.com/blog/web-performance-page-bloat/нашу добру знайому Tammy Everts</a>, але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на <a hreflang="en" href="https://blog.chriszacharias.com/page-weight-matters">вивчені десятиліття тому уроки</a>, ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```

File: src/content/uk/2020/page-weight.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+<!-- markdownlint-enable MD018 -->
+
+### Пропускна здатність
+
+У 2016, коли я попросив пояснити, чому австралійський турист, з яким я розмовляв, був у захваті від інтернету Великобританії, Ilya Grigorik з Google [сказав двома словами](https://youtu.be/x4S38hpgxuM?t=89): фізика, чорт забирай! (упс, це три).
```
Comment (tunetheweb) [inline src/content/uk/2020/page-weight.md:34]:
```suggestion
У 2016, коли я попросив пояснити, чому австралійський турист, з яким я розмовляв, був у захваті від інтернету Великобританії, Ilya Grigorik з Google <a hreflang="en" href="https://youtu.be/x4S38hpgxuM?t=89">сказав двома словами</a>: фізика, чорт забирай! (упс, це три).
```

**Review by tunetheweb — APPROVED** [review]
LGTM to me with just a few nits!

Nice to have a Ukrainian chapter.

File: src/content/uk/2020/page-weight.md (hunk 3 above)
Comment (tymosh) [reply src/content/uk/2020/page-weight.md]: My bad. Will watch links from now.

File: src/content/uk/2020/page-weight.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
...tps://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру <a hreflang="en" href="https://www.wired.com/2016/04/average-webpage-now-size-original-doom/">оригінальної Doom</a>. Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи <a hreflang="en" href="https://www.speedcurve.com/blog/web-performance-page-bloat/нашу добру знайому Tammy Everts</a>, але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на <a hreflang="en" href="https://blog.chriszacharias.com/page-weight-matters">вивчені десятиліття тому уроки</a>, ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```
Comment (tunetheweb) [inline src/content/uk/2020/page-weight.md:29]:
```suggestion
#РозмірСторінкиВсеЩеВажливий майже натякає на те, що розмір сторінки не важливий, або ніколи не був. Можливо, він був не важливим, коли запустився базований на тексті [Craigslist](https://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру <a hreflang="en" href="https://www.wired.com/2016/04/average-webpage-now-size-original-doom/">оригінальної Doom</a>. Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи <a hreflang="en" href="https://www.speedcurve.com/blog/web-performance-page-bloat/">нашу добру знайому Tammy Everts</a>, але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на <a hreflang="en" href="https://blog.chriszacharias.com/page-weight-matters">вивчені десятиліття тому уроки</a>, ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```

**Comment by tunetheweb** [comment]
Thanks very much for that translation @tymosh ! We’re holding off any releasing until we launch the full 2021 edition on the 1st December and this will be included in that.

**Pull request merged**